github3/pulls.py

```diff
@@ -795,18 +823,44 @@ def _update_attributes(self, review):
         self.id = review['id']
         self.author_association = review['author_association']
         self.body = review['body']
-        self.body_html = review['body_html']
-        self.body_text = review['body_text']
+        self.body_html = review.get('body_html')
+        self.body_text = review.get('body_text')
         self.commit_id = review['commit_id']
         self.html_url = review['html_url']
         self.user = users.ShortUser(review['user'], self)
         self.state = review['state']
-        self.submitted_at = review['submitted_at']
+        self.submitted_at = review.get('submitted_at')
```

Comment:
This is optional for PENDING state reviews.

Reply:
If this is a datetime being returned by the API, it needs to be wrapped in `self._strptime()` which, iirc, handles `None`